frontend/src/utils/CampUtils.ts

```diff
@@ -34,3 +34,26 @@ export const campStatus = (camp: CampResponse): CampStatus => {
   }
   return CampStatus.COMPLETED;
 };
+
+export const getFormattedDateString = (dates: Array<string>): string => {
+  const startDate = new Date(dates[0]).toLocaleDateString("en-us", {
+    month: "short",
+    day: "numeric",
+  });
+  const endDate = new Date(dates[dates.length - 1]).toLocaleDateString(
+    "en-us",
+    {
+      year: "numeric",
+      month: "short",
+      day: "numeric",
+    },
+  );
+
+  return `${startDate} - ${endDate}`;
+};
+
+export const getFormattedSessionDatetime = (
+  dates: Array<string>,
+  startTime: string,
+  endTime: string,
+): string => `${getFormattedDateString(dates)} | ${startTime} - ${endTime}`;
```

Comment:
The modal you rendered doesn't show AM and PM. I can't see if this is an issue with data on the db or something you forgot in the code?

Reply:
Quick guess would be backend? I don't see any parsing on the frontend :/

Reply:
Yeah backend